rpc-interface/src/blockchain.rs

```diff
@@ -97,10 +99,36 @@ pub trait BlockchainInterface {
         &mut self,
         address: Address,
         include_stakers: Option<bool>,
-    ) -> Result<Validator, Self::Error>;
+    ) -> Result<BlockchainState<Validator>, Self::Error>;
 
-    async fn get_staker_by_address(&mut self, address: Address) -> Result<Staker, Self::Error>;
+    async fn get_staker_by_address(
+        &mut self,
+        address: Address,
+    ) -> Result<BlockchainState<Staker>, Self::Error>;
+
+    #[stream]
+    async fn head_subscribe(
```

Comment:
From a RPC user perspective, I would like to propose some naming alternatives for the streaming methods:
* `head_subscribe` -> `subscribe_for_head_block`
* `head_hash_subscribe` -> `subscribe_for_head_block_hash`
* `election_validator_subscribe` -> `subscribe_for_validator_election_by_address` 
* `logs_subscribe` -> `subscribe_for_logs` or `subscribe_for_block_logs` or `subscribe_for_event_logs`
* `logs_by_type_and_addresses_subscribe` -> `subscribe_for_logs_by_addresses_and_types` or one of the other two variations mentioned above

Reply:
Thank you! 
The methods are renamed using those suggestions :)